pkg/types/price_type.go

```diff
@@ -0,0 +1,45 @@
+package types
+
+import (
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+)
+
+type PriceType string
```

Comment:
implement unmarshalJSON on PriceType ? and maybe you can transform the original string into upper case string